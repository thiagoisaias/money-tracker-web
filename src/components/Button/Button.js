import React from "react";
import styled from "styled-components";

const Container = styled.button`
  font-size: 13px;
  padding: 6px;
  border-radius: 2px;
  color: #fff;
  background-color: #add8e6;
  width: 125px;
  height: 35px;

  &:hover {
    cursor: pointer;
    background-color: #a9cdd8;
  }
`;

const Button = props => {
  return <Container {...props}> {props.children} </Container>;
};

export default Button;
