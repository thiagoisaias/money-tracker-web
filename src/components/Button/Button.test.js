import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Button from "./Button";

describe("Button", () => {
  const component = shallow(<Button />);

  it("renders properly", () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
