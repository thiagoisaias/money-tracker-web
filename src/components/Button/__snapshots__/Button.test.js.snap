// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders properly 1`] = `
<styled.button>
   
   
</styled.button>
`;
